migration/migration.go

```diff
@@ -226,15 +225,46 @@ func (m *changeTrieNodeEncoding) Before() {
 	}
 }
 
+type node struct {
+	Value *felt.Felt
+	Left  *bitset.BitSet
+	Right *bitset.BitSet
+}
+
+func (n *node) _WriteTo(buf *bytes.Buffer) (int64, error) {
```

Comment:
Will this migration be applied automatically? or it's just refactoring of old one and in runtime everything is going to be ok ?

Reply:
> or it's just refactoring of old one and in runtime everything is going to be ok ?

Yeap, it is just refactoring an old migration. It should still behave exactly the same.